models.py

```diff
@@ -728,7 +729,9 @@ class ApiKeyModelView(BaseOnlyUserOwnedModelView):
     def _format_qrcode(view, context, model, name):
         admin = model.account_admin if model.account_admin else False
         address = model.user.wallet_address if model.user.wallet_address else ''
-        data = 'zapm_apikey:%s?secret=%s&name=%s&admin=%r&address=%s' % (model.token, model.secret, model.name, admin, address)
+        url = urlparse(request.base_url)
+        server = url.scheme+'://'+url.netloc.split(":")[0]
```

Comment:
dont strip out the port part of the netloc (if there is a port specified in the url then we should use it, also the code as is will crash if there is no port specified)

also proper python string formatting (https://pyformat.info/#simple)